bentoml/_internal/models/model.py

```diff
@@ -0,0 +1,257 @@
+import logging
+import os
+import shutil
+import typing as t
+from collections import UserDict
+from datetime import datetime, timezone
+
+import attr
+import fs
+import fs.errors
+import fs.mirror
+import yaml
+from fs.base import FS
+from simple_di import Provide, inject
+
+from ...exceptions import BentoMLException
+from ..configuration import BENTOML_VERSION
+from ..configuration.containers import BentoMLContainer
+from ..store import Store, StoreItem
+from ..types import PathType, Tag
+
+if t.TYPE_CHECKING:
+    from ..runner import Runner
+
+logger = logging.getLogger(__name__)
+
+MODEL_YAML_FILENAME = "model.yaml"
+
+PathRes = t.TypeVar("PathRes", bound=t.Union[t.Optional[str], str])
```

Comment:
dead code?
if not then maybe
```suggestion
PathRes = t.Union[t.Optional[str], str]
```
as a typealias rather than a typevar?

Reply:
It's old code I forgot to delete; it used to be the generic parameter to `Bento` before I just made the syspathbento because generics in python are absolutely awful lol.